src/Controller/Api/UserController.php

```diff
@@ -178,8 +178,8 @@ public function postUserAction(Request $request)
      *
      * @Put("/user/(id}.{_format}", name="put_user")
```

Comment:
```suggestion
     * @Put("/user/{id}.{_format}", name="put_user")
```
Can you check in other places too?

Reply:
Good catch! Thank you!
sonata-project/SonataUserBundle#1202.

Reply:
This will be remove in https://github.com/sonata-project/SonataUserBundle/pull/1200

Reply:
>This will be remove in #1200

Yes.